modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/info/collection/provider/test/ManyToManyObjectRelationshipInfoCollectionProviderTest.java

```diff
@@ -159,6 +162,62 @@ public void testManyToManyObjectRelationshipRelatedInfoCollectionProvider()
 		_assertRelatedInfoCollectionProvider(
 			_objectDefinition2, objectDefinition2ObjectEntry,
 			objectDefinition1ObjectEntry1, objectDefinition1ObjectEntry2);
+
+		Map<String, ServiceRegistration<?>> serviceRegistrations =
+			_objectRelationshipLocalService.getServiceRegistrations();
+
+		String serviceRegistrationKey = StringBundler.concat(
+			_objectRelationship.getCompanyId(), StringPool.POUND,
+			_objectRelationship.getObjectRelationshipId());
+
+		ServiceRegistration<?> objectRelationshipServiceRegistration =
+			serviceRegistrations.get(serviceRegistrationKey);
+
+		objectRelationshipServiceRegistration.unregister();
+
+		serviceRegistrations.remove(serviceRegistrationKey);
+
+		serviceRegistrations =
+			_objectRelationshipLocalService.getServiceRegistrations();
```

Comment:
Why do we need to call the remove method if we execute the get method in the following line?
I think we can remove the line calling the remove method.